### ISSUE TITLE
MuJoCo training results failed to reproduce

### ISSUE THREAD

**Issue by happy-yan** [ALRhub/DIME#2]
Hello, when I tried to train 
```
python run_dime.py env_name=Humanoid-v3
```
I found that the rising trend of the curve was quite different from that reported in the paper. Are there any hyperparameters that need to be adjusted? The reward for 4.0M env steps is only 3255.

**Comment by Onur4229** [comment]
Hi happy-yan,
Thank you for your question. Regarding the gym environments, the hyperparameter set for the boundaries for the distributional critic is 
` v_min=-1600, v_max=1600` 
as stated on page 18 in the paper. 
This means you will need to overwrite the default values e.g. by
` python run_dime.py env_name=Humanoid-v3  alg.critic.v_min=-1600 alg.critic.v_max=1600`  

Depending on which gym version you would like to use, you might also need to wrap the gym environment during creation using
`training_env = GymV21CompatibilityV0(env_id=cfg.env_name)`  
The necessary import for this is in shimmy:
`from shimmy import GymV21CompatibilityV0`

I hope this resolves your problem. Please let me know if you have any other questions.

**Comment by happy-yan** [comment]
thank you i will try it

**Comment by Onur4229** [comment]
Hi happy-yan,
Thanks for your reply. Did it work out? In this case, I will close the issue.

**Comment by happy-yan** [comment]
Thank you ~
